Tests/SwiftKafkaTests/KafkaConsumerTests.swift

```diff
@@ -85,4 +87,92 @@ final class KafkaConsumerTests: XCTestCase {
             )
         }
     }
+
+    func testConsumerStatistics() async throws {
+        // Set no bootstrap servers to trigger librdkafka configuration warning
+        let uniqueGroupID = UUID().uuidString
+        var config = KafkaConsumerConfiguration(
+            consumptionStrategy: .group(id: uniqueGroupID, topics: ["this-topic-does-not-exist"])
+        )
+        config.statisticsInterval = Duration.milliseconds(100)
+
+        let stringJson = NIOLockedValueBox<String>(String())
+        let consumer = try KafkaConsumer(config: config, logger: .kafkaTest)
+
+        let statTask = Task {
```

Comment:
This can be part of the TaskGroup I think

Reply:
Oh, certainly yes, thank you for the hint!
Should we probably collect all tasks from the group as they all should be finished? 
Like this:
```
for try await _ in group { }
```

Reply:
You invoke `group.next()` to receive the result of the next completed task. You don't have to invoke it for all of the Tasks you add. When you take a look at the `SwiftKafkaTests.swift` you can see that I used `group.next()` to wait for the producer/consumer task to be finished so I can safely `triggerGracefulShutdown`  _after_ all my producer/consumer testing logic has run.

[Documentation for reference](https://developer.apple.com/documentation/swift/taskgroup/next())